src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -46,6 +47,52 @@ namespace
 
 namespace JSONHelpers
 {
+    bool isValidGuid(const std::wstring& str)
+    {
+        GUID id;
+        return SUCCEEDED_LOG(CLSIDFromString(str.c_str(), &id));
+    }
+
+    bool isValidDeviceId(const std::wstring& str)
+    {
+        std::wstring temp;
+        std::vector<std::wstring> parts;
```

Comment:
Maybe this validation function would be more readable if instead of parts array we used actual names for those elements inside device id, which are device id, monitor width, monitor height and virtual desktop id. Or at least comment on the format inside this function.